src/stdune/option.mli

```diff
@@ -22,6 +22,8 @@ val forall: 'a t -> f:('a -> bool) -> bool
 val value : 'a t -> default:'a -> 'a
 val value_exn : 'a t -> 'a
 
+val value_map : 'a t -> f:('a -> 'b) -> default:'b -> 'b
```

Comment:
this function does not seem to be used

Reply:
Ups, indeed I refactored code and it is not used anymore, thanks!